File: PrimeNumberDeatils/Prime.kt

fun isPrime(n: Int): Boolean {
    if (n <= 1) return false
    for (i in 2..Math.sqrt(n.toDouble()).toInt()) {
        if (n % i == 0) return false
    }
    return true
}

fun main() {
    print("Enter a number: ")
    val num = readLine()!!.toInt()
    if (isPrime(num))
        println("\$num is a prime number.")
    else
        println("\$num is not a prime number.")
}

File: PrimeNumberDeatils/prime.rb

def is_prime(n)
    return false if n <= 1
    (2..Math.sqrt(n)).each do |i|
        return false if n % i == 0
    end
    true
end

print "Enter a number: "
num = gets.to_i
if is_prime(num)
    puts "#{num} is a prime number."
else
    puts "#{num} is not a prime number."
end
